sdk/communication/azure-communication-identity/dev_requirements.txt

```diff
@@ -3,4 +3,5 @@
 ../../core/azure-core
 ../azure-mgmt-communication
 aiohttp>=3.0; python_version >= '3.5'
--e ../../../tools/azure-devtools
\ No newline at end of file
+-e ../../../tools/azure-devtools
+parameterized>=0.7.3
```

Comment:
not sure what is this. can you pls verify?

Reply:
this library for testing which provides features running tests with different value of the parameters